app/glossary.md

```diff
@@ -10,7 +10,7 @@ A collection of terms commonly used across the Becoming a Teacher service lines.
 Accreditation
 : All initial teacher training providers must be ‘accredited’ by the Secretary of State for Education. This process of ‘accreditation’ confirms they have permission to offer courses leading to Qualified Teacher Status (QTS). (NB: Ofsted is the body which checks the courses providers offer meet the standards set by the department.)
 : The exception is the School Direct (SD) training route; SDs need a ‘ratifying accredited provider’. The ratifying accredited provider may be a School Centred Initial Teacher Training (SCITT) or a Higher Education Institute (HEI), and is responsible for maintaining the quality of the training provided.
-: As accredited providers, SCITTs can offer QTS courses but do not have ‘degree awarding powers’. So to cater to candidates seeking academic awards (for example, a Post Graduate Certificate in Education (PGCE)), they need to partner with an HEI.
+: As accredited providers, SCITTs can offer QTS courses but do not have ‘degree awarding powers’. So to cater to candidates seeking academic awards (for example, a Post Graduate Certificate in Education (PGCE), they need to partner with an HEI.
```

Comment:
That second closing bracket is needed - read the sentence again! Maybe this can be reworded to avoid nested bracketing?